### ISSUE TITLE
Using autobox in a docking search 

### ISSUE THREAD

**Issue by rwxayheee** [ccsb-scripps/AutoDock-Vina#265]
Dear all, 

I realized from issue #264 that `--autobox` doesn't set grid size or center in a docking search. But Vina will print the following to standard output whenever it receives `--autobox`, which I feel a little misleading: 

`Output will be ligand_out.pdbqt
Scoring function : vina
Rigid receptor: receptor.pdbqt
Ligand: ligand.pdbqt
Grid center: ligand center (autobox)
Grid size  : ligand size + 4 A in each dimension (autobox)
Grid space : 0.375`

In a docking serach (not `score_only` or `local_only`), `--size_x --size_y --size_z` are required with and without `--autobox`. `--center_x --center_y --center_z` are optional, but if unspecified it will be 0,0,0. The true grid size and center are not noted in the standard output, when `--autobox` is used. 

It might be helpful to change the standard output, or reject `--autobox` if using it in docking search is not intended. 

Version of the program: 
[vina_1.2.5_linux_x86_64](https://github.com/ccsb-scripps/AutoDock-Vina/releases/download/v1.2.5/vina_1.2.5_linux_x86_64)

I'm running it on: Linux x86_64 (cpe:/o:redhat:enterprise_linux:7.9:GA:server)

Any comments would be much appreciated. Thank you for your time and the excellent work you've done.

**Comment by diogomart** [comment]
Yeah we should reject `--autobox` if used in a docking search. That was the original intent as per the help message:
```
  --autobox                  set maps dimensions based on input ligand(s) (for 
                             --score_only and --local_only)
```
Thank you for looking into this.

**Comment by rwxayheee** [comment]
I would like to add this to Maintenance Release, if it's convenient to make some minor improvement to option handling
